CHANGELOG.md

```diff
@@ -35,6 +35,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [v0.39.1]
+
+### Client Breaking Changes
+
+* (x/auth) [\#6861](https://github.com/cosmos/cosmos-sdk/pull/6861) Remove public key Bech32 encoding for all account types for JSON serialization, instead relying on direct Amino encoding. In addition, JSON serialization utilizes Amino instead of the Go stdlib, so integers are treated as strings.
```

Comment:
If we use amino codec to json encode all vesting accounts, shall we also do that for the base account?

I guess we would need a custom Marshall, but without bech32.

It would be good to have some test cases showing the actual json output. And assert such a struct marshals like that. Not just for tests, but living docs for the api - as this is a central and changing structure

Reply:
> If we use amino codec to json encode all vesting accounts, shall we also do that for the base account?
>
> I guess we would need a custom Marshall, but without bech32.

No. `BaseAccount` already uses Amino as it does _not_ have custom marshaling.

Reply:
Ahhh.... default is amino not encoding/json.

Fair enough, this all makes sense to me. Also a comment from Simon that the account number went from string in 0.37 to number in 0.38 to string in 0.39. So we are back to 0.37 behavior :+1: